build/psr12/psr12_converter.php

```diff
@@ -0,0 +1,242 @@
+
+
+        TEXT;
+    die(1);
+}
+
+foreach($argv as $arg) {
+    if (substr($arg, 0, 2) === '--') {
+        $argi = explode('=', $arg, 2);
+        switch($argi[0]) {
+            case '--task':
+                foreach($tasks as $task=>$value) {
+                    if (stripos($argi[1], $task) !== false) {
+                        $tasks[$task] = true;
+                    }
+                }
+                break;
+        }
+    } else {
+        $checkPath = $arg;
+        break;
+    }
+}
+
+if ($tasks['CMS']) {
+    $tasks['CBF']   = true;
+    $tasks['CS']    = true;
+    $tasks['CLEAN'] = true;
+}
+
+$items = [];
+if ($checkPath) {
+    $items = explode(',', $checkPath);
+} else {
+    $items[] = 'index.php';
+    $items[] = 'administrator/index.php';
+
+    $baseFolders = [
+        'administrator/components',
+        'administrator/includes',
+        'administrator/language',
+        'administrator/modules',
+        //'administrator/templates',
+        'api',
+        'cli',
+        'components',
+        'includes',
+        'installation',
+        'language',
+        // 'layouts',
+        'libraries',
+        'modules',
+        'plugins',
+        // 'templates',
+    ];
+
+    foreach ($baseFolders as $folder) {
+        $dir = dir($root . '/' . $folder);
+        while (false !== ($entry = $dir->read())) {
+            if (($entry === ".") || ($entry === "..")) {
+                continue;
+            }
+            if (!is_dir($dir->path . '/' . $entry)) {
+                if (substr($entry, -4) !== '.php') {
+                    continue;
+                }
+            }
+            $items[] = str_replace($root .'/', '',$dir->path) . '/' . $entry;
+        }
+        $dir->close();
+    }
+}
+$executedTasks = implode(',', array_keys(array_filter($tasks,function($task) { return $task; } )));
+$executedPathes = implode("\n", $items);
+
+echo <<<TEXT
+        Joomla! PSR12 Converter
+        =======================
+
+        Tasks will be executed: {$executedTasks}
+        Files and Folders:
+        {$executedPathes}
+
+
+        TEXT;
+
+// Recreate temp dir
+$cleanItems=glob($tmpDir.'/{,.}*',GLOB_MARK | GLOB_BRACE);
+foreach($cleanItems as $item)
+{
+    if(basename($item)=='.' || basename($item)=='..') {
+        continue;
+    }
+    unlink($item);
+}
+unset($cleanItems, $item);
+
+@mkdir($tmpDir, 0777, true);
+
+$php        = 'php';
+$git        = 'git';
+$cbfOptions = "-p --standard=" . __DIR__ . "/ruleset.xml --extensions=php";
+$csOptions  = "--standard=" . __DIR__ . "/ruleset.xml --extensions=php";
+$csOptions  .= " --report=$root/build/psr12/phpcs.joomla.report.php";
+
+foreach ($items as $item) {
+    if ($tasks['CBF']) {
+        echo 'Fix ' . $dir->path . '/' . $entry . "\n";
+
+        passthru($php . ' ' . $root . '/libraries/vendor/bin/phpcbf ' . $cbfOptions . ' ' . $item, $result);
+
+        if ($result !== 0) {
+            echo "Error PHPCBF complete with error code: $result \n\n";
```

Comment:
not sure if this actually completes the task or exits the task so it should be either

```suggestion
            echo "Error PHPCBF completed with error code: $result \n\n";
```

or it should be
```suggestion
            echo "Error PHPCBF exited with error code: $result \n\n";
```

and the same for all the other error messages below